Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -8,33 +8,51 @@ class Lesson3
     {
         public static void Demo()
         {
-            Console.WriteLine("Temp, not sure what to do here yet");
+            // Person 1 Input (name, age, weight, height)
```

Comment:
Use function for bmi processing as well. Now you have 2 duplicate blocks of code which can be perfectly extracted to a function.

Reply:
Wow that's very smart! I didn't even think of that!